scripts/skills/injury/rf_black_eye_injury.nut

```diff
@@ -0,0 +1,73 @@
+this.rf_black_eye_injury <- this.inherit("scripts/skills/injury/injury", {
+	m = {
+		VisionModifier = -1,
+		MeleeSkillMult = 0.9,
+		RangedSkillMult = 0.9,
+	},
+	function create()
+	{
+		this.injury.create();
+		this.m.ID = "injury.rf_black_eye";
+		this.m.Name = "Black Eye";
+		this.m.Description = "A powerful blow to the eye socket left the area bruised, swollen, and discolored.";
+		this.m.Type = this.m.Type | ::Const.SkillType.TemporaryInjury;
+		this.m.DropIcon = "rf_black_eye_injury";
+		this.m.Icon = "ui/injury/rf_black_eye_injury.png";
+		this.m.IconMini = "rf_black_eye_injury_mini";
+		this.m.HealingTimeMin = 1;
+		this.m.HealingTimeMax = 3;
+	}
+
+	function getTooltip()
+	{
+		local ret = this.skill.getTooltip();
+
+		if (this.m.VisionModifier != 0)
+		{
+			ret.push({
+				id = 10,
+				type = "text",
+				icon = "ui/icons/vision.png",
+				text = ::MSU.Text.colorRed(this.m.VisionModifier) + " Vision"
```

Comment:
We can take this opportunity to add `Vision` as a concept. To do this you can add a commit to the `z_nested_tooltips.nut` file and in the `AutoConcepts` array you can add `"character-stats.SightDistance"`. Then here it can be parsed using `[Vision|Concept.SightDistance]`